multiqc/modules/biscuit/biscuit.py

```diff
@@ -0,0 +1,688 @@
+            sid = sid.replace('.bam','')
+            sid = sid.replace('_markdup','')
+            if sid not in pd:
+                pd[sid] = {}
+            if 'dupRatePE' in dd and dd['dupRatePE'] is not None:
+                pd[sid]['%dupratePE'] = dd['dupRatePE']
+                hasPE = True
+            if 'dupRateSE' in dd and dd['dupRateSE'] is not None:
+                pd[sid]['%duprateSE'] = dd['dupRateSE']
+                hasSE = True
+
+        pheader = OrderedDict()
+        pheader['%aligned'] = {'title':'% Aligned', 'max':100, 'min':0, 'suffix':'%','scale':'Greens'}
+        if hasPE:
+            pheader['%dupratePE'] = {'title':'% DupsPE', 'max':100, 'min':0, 'suffix':'%','scale':'Reds'}
+        if hasSE:
+            pheader['%duprateSE'] = {'title':'% DupsSE', 'max':100, 'min':0, 'suffix':'%','scale':'Reds'}
+        self.general_stats_addcols(pd, pheader)
+
+        # if len(mdata)>0 and all([dd['Duplication Rate'] is not None for dd in mdata.values()]):
+        # self.general_stats_addcols(mdata, {'% Dup': {'suffix':'%', 'min':0, 'max':100}})
+
+
+    def parse_logs_mapq(self, f, fn):
+
+        data = {}
+        if fn.endswith('_mapq_table.txt'):
+            for l in f.splitlines():
+                s = l.split()
+                data[s[0]] = s[1]   # mapping quality > number of reads
+                
+        if fn.endswith('_strand_table.txt'):
+            for l in f.splitlines():
+                m = re.search(r'strand\t([+-]*)\t(\d+)', l)
+                if m is not None:
+                    data[m.group(1)] = int(m.group(2))
+
+        if fn.endswith('_isize_score_table.txt'):
+            data['I'] = {}      # insert size
+            data['S'] = {}      # SW-score
+
+            for l in f.splitlines():
+                fields = l.split()
+                if fields[0] == 'I':
+                    data[fields[0]][int(fields[1])] = float(fields[2]) * 100
+                elif fields[0] == 'S':
+                    data[fields[0]][fields[1]] = float(fields[2])
+                
+        return data
+    
+    def chart_biscuit_mapq(self):
+
+        # fraction of optimally mapped reads
+        pd = {}
+        for sid, dd in self.mdata['mapq'].items():
+            if not sid.endswith('_mapq_table.txt'):
+                continue
+            sid = sid.replace('_mapq_table.txt','');
+            pd[sid] = {'OAligned':0, 'SAligned':0, 'UAligned':1}
+            for mapq, cnt in dd.items():
+                if mapq == 'unmapped':
+                    pd[sid]['UAligned'] += int(cnt)
+                elif int(mapq) >= 40:
+                    pd[sid]['OAligned'] += int(cnt)
+                else:
+                    pd[sid]['SAligned'] += int(cnt)
+
+        self.add_section(
+            name = 'Mapping Summary',
+            anchor = 'biscuit-mapping',
+            plot = bargraph.plot(pd, OrderedDict([
+                ('OAligned', {'name':'Optimally Aligned Reads', 'color': '#8bbc21'}),
```

Comment:
Personally, I prefer charts to use colours from the existing MultiQC colour palette (the defaults here should actually be pretty good I think? Blue / Dark Blue / Orange off the top of my head). But it's your code, so if you really prefer these colours then it's fine to stick with them 😉